src/PIL/ImageShow.py

```diff
@@ -136,7 +136,7 @@ class WindowsViewer(Viewer):
     """The default viewer on Windows is the default system application for PNG files."""
 
     format = "PNG"
-    options = {"compress_level": 1}
+    options = {"compress_level": 1, "save_all": True}
```

Comment:
Photos, the default Windows viewer for images, does not support APNG and only shows the first frame. However, users can change their default viewer to one that does support APNG, so this is still a worthwhile change.